app/views/shared/_geo_data_info.html.erb

```diff
@@ -0,0 +1,14 @@
+<ul class="list-inline">
+  <li><%= metadata['contentType']&.each { |type| type['value'] }&.join(' ; ') %></li>
```

Comment:
I noted this in slack, but I think the css rules need to be updated for these `<li>` tags to include something like:

`vertical-align: top;`

to address weird alignment issues.